Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcServer.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Blizzard.IO.Core.Rpc;
+using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace Blizzard.IO.RabbitMQ.Rpc
+{
+    public class NetqRabbitRpcServer : IRpcServer
+    {
+        private readonly INetqRpcRabbitConnection<Func<Type,object>> _netqRpcRabbitConnection;
+        private readonly ILogger _logger;
+
+        private IDisposable _respondHandler;
```

Comment:
Maybe i missed something but i didn't understand what you do with the responseHandler?

Reply:
I use it to stop the server from responding without disposing the whole bus (and than terminating the other servers and clients that uses the same bus)